sandbox/vertical_container.py

```diff
@@ -0,0 +1,94 @@
+from rich.console import RenderableType
+from rich.text import Text
+
+from textual.app import App, ComposeResult
+from textual.widget import Widget
+from textual.widgets import Placeholder
+
+root_container_style = "border: solid white;"
+initial_placeholders_count = 4
+
+
+class VerticalContainer(Widget):
```

Comment:
a quick "vertical layout" sandbox I created to reproduce the bug I was working on

https://user-images.githubusercontent.com/722388/167122123-8ddfe75f-4588-4c4f-97ea-e891143d94ae.mp4